.github/workflows/actions.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: 'flutter/flutter'
-          ref: 'stable'
+          ref: 'beta'
```

Comment:
I noticed now too that the name of this step needs to be renamed to:

`Clone Flutter repository with beta channel`

Or maybe just:

`Clone Flutter repository`

Reply:
Done. Nice catch!